docs/man/docker-run.1.md

```diff
@@ -110,6 +111,14 @@ milliseconds.
 **--cidfile**=""
    Write the container ID to the file
 
+**--cpu-quota**=0
+   CPU cfs quota
+
+   You can limit the cpu usage of a container 
+with the --cpu-quota option. By default, every container run with full cpu resource,
+but you can tell the kernel to limit the usage of a container when you start then 
+via **docker run**
```

Comment:
Limits the container's CPU usage. By default, containers run with the full CPU resource.  This flag causes the kernel to restrict the container's CPU usage to the value you specify.